src/main/java/com/finance/app/infrastructute/security/FinanceAuthHandler.java

```diff
@@ -0,0 +1,63 @@
+package com.finance.app.infrastructute.security;
+
+import com.finance.app.constant.FinanceConstants;
+import com.finance.app.domain.enums.UserStatus;
+import lombok.Getter;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.authentication.AuthenticationSuccessHandler;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+
+public class FinanceAuthHandler implements AuthenticationSuccessHandler {
+
+    private final static String ACTIVE = "ACTIVE";
+    private final static String UNDER_VERIFICATION = "UNDER_VERIFICATION";
+    private final static String SUSPENDED = "SUSPENDED";
```

Comment:
I already seen such enum , why do you duplicate the date ? Reuse enum to contain such values.